wikidatarefisland/run.py

```diff
@@ -44,6 +44,10 @@ def main(argv, filepath):
         the_scraper = pipes.ScraperPipe(config, schemaorg_normalizer, schemaorg_mapper)
         simple_pump.run(the_scraper, args.input_path, args.output_path)
 
+    if 'itemextract' == args.step:
```

Comment:
Nitpick: I think we can say something like "extract_items"

Reply:
I also would put it at above the pipe2 since it's pipe1 but nitpick :D